src/MessagingAttachment/MessagingAttachment.less

```diff
@@ -162,6 +161,72 @@
   }
 }
 
+.MessagingAttachment--ParentContainer.MessagingAttachment--ParentContainer--IsUpload {
+  overflow: visible; // ensures you can see the x icon in top right corner
+}
+
+// Family portal specific styling
+.MessagingAttachment--ParentContainer.MessagingAttachment--FamilyPortal {
+  border-radius: @size_m;
+  background-color: @neutral_white;
+  box-shadow: @size_none @size_3xs 0.375rem rgba(71, 76, 94, 0.3);
```

Comment:
Looks like we can remove this line now since we have the `boxShadow--medium()` on the next line

Reply:
🤦